### PR TITLE
CU-862j2km6g_Dialog

### PR DESCRIPTION
**Pull request by kurylek**
Added Dialog component.
You have to pass 5 attributes:

- dialogText- Text to display in dialog
- confirmLabel- Confirm button label
- confirmAction- Action on submit
- cancelAction- Action on cancel
- inputsInfo- Information about inputs in table; Input is defined by: inputName, inputType, inputLabel

```
function createInput(inputName: string, inputType: string, inputLabel: string) {
  return { inputName, inputType, inputLabel };
}
```

### CODE REVIEW

**Comment by s20146** [comment]
Task linked: [CU-862j2km6g Dialog](https://app.clickup.com/t/862j2km6g)

File: React/src/components/Dialog/Dialog.tsx [hunk 1]
```diff
@@ -0,0 +1,92 @@
...
+  }));
+
+const Dialog = ({ dialogText, confirmLabel, inputsInfo, confirmAction, cancelAction }: DialogTypes) => {
+    const inputsHeight = 90 * inputsInfo.length;
+    const dialogHeight = 110;
+    const totalHeight = (inputsHeight+dialogHeight)+"px";
+    
+  return (
+    <div className="blur-background">
+        <div className="dialog" style={{height: totalHeight}}>
```
Comment (Marcin-H) [inline React/src/components/Dialog/Dialog.tsx:27]: I think we should use MUI Components, not html elements. We could use for example Box or Container or Grid

File: React/src/components/Dialog/Dialog.tsx [hunk 2]
```diff
@@ -0,0 +1,92 @@
...
+            <form>
+                <p>
```
Comment (Marcin-H) [inline React/src/components/Dialog/Dialog.tsx:29]: text should be added as <Typography /> if we are using mui

**Review by Marcin-H — APPROVED** [review]
We can discuss with frontend group how we should create structure. Should we use only Mui components or can we use normal html elements

**Pull request merged**